### ISSUE TITLE
nl2br not working with version 4.0.0 and linkify-html

### ISSUE THREAD

**Issue by lukasvice** [Hypercontext/linkifyjs#403]
Does the `nl2br` option not work with version 4.0.0. and `linkify-html` anymore?
The expected output should be the output from version 3.0.5. See below.

## Reproduction

Script:
```js
// version 4.0.0
// const linkifyHtml = require('linkify-html')

// version 3.0.5
const linkifyHtml = require('linkifyjs/html')

const text = `E-Mail: info@example.com.
Web: www.example.com`

const html = linkifyHtml(text, { nl2br: true })

console.log(html)

```
## Output

version 3.0.5
```
E-Mail: <a href="mailto:info@example.com">info@example.com</a>.<br>Web: <a href="http://www.example.com">www.example.com</a>
```

version 4.0.0 with `linkify-html`
```
E-Mail: <a href="mailto:info@example.com">info@example.com</a>.
Web: <a href="http://www.example.com">www.example.com</a>
```

**Comment by nfrasser** [comment]
Hi @lukasvice, thanks for reporting. This should be fixed in the latest v4.0.1 release.

**Comment by lukasvice** [comment]
Thank you! 🎉
